rust/stracciatella/src/schemas/yaml/strategic-mines.schema.yaml

```diff
@@ -14,10 +14,10 @@ items:
       title: Entrance Sector
```

Comment:
can you split this one into the previous commits, so they'll be buildable as well?